lib/patch-series.ts

```diff
@@ -174,27 +176,33 @@ export class PatchSeries {
             metadata.coverLetterMessageId = "not yet sent";
         }
 
-        const {
-            basedOn,
-            cc,
-            coverLetter,
-        } = PatchSeries.parsePullRequestDescription(pullRequestDescription);
+        const parsedPR =
+            PatchSeries.parsePullRequestDescription(pullRequestDescription);
 
-        if (basedOn && !revParse(basedOn, workDir)) {
-            throw new Error(`Cannot find base branch ${basedOn}`);
+        if (parsedPR.basedOn && !revParse(parsedPR.basedOn, workDir)) {
+            throw new Error(`Cannot find base branch ${parsedPR.basedOn}`);
+        }
+
+        let coverLetter = pullRequestTitle + "\n\n" + parsedPR.coverLetter;
+
+        const wrapCoverLetterAtColumn = 76;
+        coverLetter = md2text(coverLetter, wrapCoverLetterAtColumn);
+
+        // if known, add submitter to email chain
+        const cc = parsedPR.cc || [];
+        if (pullRequestSubmitter.length) {
+            cc.push(pullRequestSubmitter);
```

Comment:
Hmm, my lack of Typescript understanding doesn't allow me to see the bug.

Trying to read code with general sense derived from C++, I see the following:

1. `parsePullRequestDescription` has a local variable that it later ruturns. It can be either `string[]` or `undefined`.
2. `getFromNotes` calls `parsePullRequestDescription` and receives result into `cc`.
3. At this time, I want to add one more item, so I have to ensure that it is no longer `undefined`, so I allocate a new `[]` if needed.

I can't understand where I made a mistake. Also, what do you mean by "third parameter to parsePullRequestDescription()" ?

Reply:
It was too late at night, I misread `parsePullRequestDescription()` to take `cc` as an optional parameter. But that optional parameter (that does not exist yet) would Just Make Sense in the context of this PR.